spec/support/helper_methods.rb

```diff
@@ -54,11 +58,7 @@ def with_env(new_env)
         yield
       ensure
         old_env.each do |key, value|
-          if value == false
-            ENV.delete key
-          else
-            ENV[key] = value
-          end
+          value == false ? ENV.delete(key) : ENV[key] = value
```

Comment:
A little shorter. But is it _too_ short???

```suggestion
          value ? ENV[key] = value : ENV.delete(key)
```

Reply:
I thought about that, but then I wondered: why did whoever wrote this tested specifically against `false`?
Since `value` can be anything, changing the implementation to the shorter one you suggested will cause the condition to be extended to all `falsy` values.
For example, if `value` is `0` then it will be deleted, but under the current implementation it would remain

Reply:
`0` is truthy in ruby, but your point still stands because of `nil`.